### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 10

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#186] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152252994-f980cdc0-5237-428c-baf6-d2c57f6e2b30.png)

**Comment by Exyss** [comment]
$\binom{18}{6} \binom{12}{6} \binom{6}{6}$

_Stesso approccio risolutivo usato per gli esercizi #8, #12_

**Reply by Dieal** [reply]
> ( 18 6 ) ( 12 6 ) ( 6 6 )
> 
> _Stesso approccio risolutivo usato per gli esercizi #8, #12_

Credo ci sia anche bisogno di dividere per 3!, in quanto ci potrebbero essere ripetizioni dovute all'ordine.
Questa è la soluzione del prof allo stesso esercizio ma con valori diversi

![SmartSelect_20250116-223904_ReadEra](https://github.com/user-attachments/assets/16966b6c-bc7b-4da8-883e-820fb372ff8b)

**Reply by Exyss** [reply]
Confermo
